### DISCUSSION TITLE
Vietnamese Unicode cannot be display properly in wiki pages.

### DISCUSSION THREAD

**Discussion by tctruc79** [requarks/wiki#3630] [Error / Bug Report]
**Describe the bug**
Vietnamese Unicode cannot be display properly in wiki pages.

**To Reproduce**
1. Create a new Page in Vietnamese Unicode (both Markdown and HTML pages), the display before saving is ok.
2. Click on 'Save' to save the page.
3. Refresh the page.
4. All 2-byte Unicode characters are changed to '?' except the page title.

**Expected behavior**
After saving Vietnamese Unicode pages, the Unicode should be displayed properly.  

**Screenshots**
![image](https://user-images.githubusercontent.com/36126386/99370846-c22e2c00-28f0-11eb-9e34-3d4b166a651d.png)If 

**Host Info (please complete the following information):**
 - OS: Docker Container (Linux)
 - Wiki.js version: 2.5.170
 - Database engine: Microsoft SQL Azure (RTM) - 12.0.2000.8

**Additional context**

**Comment by p1nkv4lkyr13** [comment]
@tctruc79 you may need to check the data type of column **pages.content** in your MSSQL database. If it is **varchar(MAX)** then change it to **nvarchar(MAX)** to make it able to save Unicode character.
